src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -287,6 +287,11 @@ namespace winrt::TerminalApp::implementation
             _defaultPointerCursor = CoreWindow::GetForCurrentThread().PointerCursor();
         }
         CATCH_LOG();
+
+        if (CascadiaSettings::IsDefaultTerminalAvailable())
```

Comment:
I wanted to add a check here that only makes it show up if conhost is the current default:
```cpp
if (CascadiaSettings::IsDefaultTerminalAvailable() && !_settings.CurrentDefaultTerminal())
```

But unfortunately the `Name()` returned by `CurrentDefaultTerminal()` is always a localized name of the application including conhost. So I'd have to add some new API functions to make this work.

But before I do that I wanted to ask if anyone else would like to see that...

Reply:
we will probably want to hide this infobar if we're not running packaged, also. which is silly.

Reply:
Shouldn't we just change `IsDefaultTerminalAvailable` to detect unpackaged activation then?